arch/arm/boot/dts/overlays/overlay_map.dts

```diff
@@ -450,4 +450,13 @@
 	vl805 {
 		bcm2711;
 	};
+
+	w1-gpio {
```

Comment:
An omitted platform is an unsupported platform, so both non-pi5 overlays also need:
```
		bcm2835;
		bcm2711;
```

Reply:
Sorry - I must have imagined these were already there: Can you add them?:
```
       w1-gpio-pi5 {
               bcm2712;
       };

       w1-gpio-pullup-pi5 {
               bcm2712;
       };
```